config/routes.rb

```diff
@@ -2,60 +2,81 @@
 
 # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 Rails.application.routes.draw do
-  resources :app_settings, only: %i[update edit]
-  resources :operating_systems, path: '/vms/requests/operating_systems', except: :show
+  # You can specify what Rails should route '/' to with the root method
+  # https://guides.rubyonrails.org/routing.html#using-root
+  root 'landing#index'
 
-  resources :request_templates, path: '/vms/request_templates', except: :show
-  patch '/vms/requests/reject', to: 'requests#reject', as: 'reject'
-  resources :requests, path: '/vms/requests' do
-    post :push_to_git, on: :member
-  end
+  # '/dashboard'
+  get '/dashboard' => 'dashboard#index', as: :dashboard
```

Comment:
Do we need a dashboard controller or is a dashboard method inside the application controller also suitable?

Reply:
A dashboard method inside the application controller would totally work. The `app/controllers/dashboard_controller.rb` currently contains, in addition to the `index` methods, some further private methods, e.g. `initialize_vm_categories` and `filter_vm_categories`, which seem duplicated in `app/controllers/vms_controller.rb`. I'd rather not do this refactoring in this PR. I've opened another issue for this.

Reply:
top